README.md

```diff
@@ -22,7 +22,7 @@ To get started, first, [obtain an API token](https://appcenter.ms/settings/apito
 ```ruby
 appcenter_upload(
   api_token: "<appcenter token>",
-  owner_name: "<your appcenter account name>",
+  owner_name: "<your appcenter account username>",
```

Comment:
It can also take App Center org name, so it would be better to say:
`<your appcenter account name (user or organization)>`

Reply:
Thank you for this information! I didn't know. I adjusted the text.

Reply:
If an app is created under **your** account and you have invited collaborators with permission to create builds (i.e. the Developer role) then from **their** perspective it should read `owner_name: "<the appcenter account username that created the app>",`

Reply:
Yeah agree with that. `your` can be misleading here

Reply:
@antonys1 @Noroxs I made another attempt at improving the description, let me know what you think!

Reply:
@lumaxis Spot on, thanks

Reply:
@lumaxis I like it, but I think the `username` should be explicitly listed instead of the `name`.

`"<appcenter account name of the owner of the app (username or organization)>"`

The only reason why I created this pull request was that I stumbled upon this naming issue. Or is it possible to use the `name` instead of `username`?

Reply:
@Noroxs The problem with that is that we unfortunately name this a bit inconsistently.
- In the UI, we call the decorative, free-text name `"Name:"`. In the API requests and responses, this is `display_name`
- In the UI, we call the programmatic, URL-friendly name `"Username:"`. In the API requests and responses, this is `name`.

I took another try at improving this, maybe this is a good compromise for the moment.

Reply:
Approved 👍